### PR TITLE
Fix evaluate() printed values being incorrect by default

### PR DESCRIPTION
**Pull request by Carath**
Temporary fix for most users to [this issue](https://github.com/keras-team/keras/issues/20788) which was not present in previous versions of Keras.

This changes to 2 the default verbosity level of the model evaluate() method, as to not print incorrect values for e.g. the loss and accuracy (mismatching correct values printed during training or values computed "by hand").

This does not fix the problem as to why ``` verbose=1 ``` makes evaluate() print incorrect values while still returning correct ones when passing ``` return_dict=True ```.

### CODE REVIEW

**Comment by fchollet** [comment]
If you want to investigate the reason for the discrepancy between the final value printed by `evaluate()` in the progress bar, and the values returned by `evaluate()` (which are the correct ones), you are welcome to do so. I'm thinking it might be the progress bar callback being delayed by one batch or something like that.

However disabling logging is not the way to do it.
